python_on_whales/components/service/cli_wrapper.py

```diff
@@ -119,6 +119,8 @@ def update(
         detach: bool = False,
         force: bool = False,
         image: Optional[str] = None,
+        quiet: bool = False,
+        replicas: Optional[int] = None,
```

Comment:
Placing them here would break backward compatibility in a silent manner for people who use positional arguments. Could you add them at the end, after the argument `with_registry_authentication`? Many thanks!

Reply:
You're right, fixed!